projects/demo/src/modules/directives/copy-processor/copy-processor.component.ts

```diff
@@ -0,0 +1,24 @@
+import {Component} from '@angular/core';
+import {changeDetection} from '@demo/emulate/change-detection';
+
+import {default as example1Html} from '!!raw-loader!./examples/1/index.html';
+import {default as example1Ts} from '!!raw-loader!./examples/1/index.ts';
+import {default as exampleImportModule} from '!!raw-loader!./examples/import/import-module.txt';
+import {default as exampleInsertTemplate} from '!!raw-loader!./examples/import/insert-template.txt';
+
+import {FrontEndExample} from '../../interfaces/front-end-example';
+
+@Component({
+    selector: 'example-tui-copy-processor',
+    templateUrl: './copy-processor.template.html',
+    changeDetection,
+})
+export class ExampleTuiCopyProcessorComponent {
+    readonly example1: FrontEndExample = {
```

Comment:
Please use the new way of inline imports and `TuiDocExample`. Example:
https://github.com/TinkoffCreditSystems/taiga-ui/blob/main/projects/demo/src/modules/components/carousel/carousel.component.ts#L41-L45

Reply:
changed